packages/storybook8/stories/Concepts/AddingSounds/Docs.mdx

```diff
@@ -28,7 +28,10 @@ Recommended reading for this is found here with these common bundlers:
     <a href="https://webpack.js.org/guides/asset-management/#loading-images">webpack</a>
   </li>
   <li>
-    <a href="https://create-react-app.dev/docs/adding-images-fonts-and-files/">create-react-app</a>
+    <a href="https://react.dev/learn/build-a-react-app-from-scratch">Create a react app from scratch</a>
+  </li>
+  <li>
+    <a href="https://vite.dev/guide/#scaffolding-your-first-vite-project">Create your first project with Vite</a>
```

Comment:
I think we should keep the same since it's more referencing asset than starting a new app
```
<a href="https://create-react-app.dev/docs/adding-images-fonts-and-files/">manage assets for create-react-app</a>
<a href="https://vite.dev/guide/assets#importing-asset-as-url">manage assets for vite</a>
```

Similarly for MeetingReactions/Doc.mdx